climpred/prediction.py

```diff
@@ -145,13 +145,14 @@ def _get_metric_comparison_dim(initialized, metric, comparison, dim, kind):
         else PROBABILISTIC_PM_COMPARISONS
     )
     if metric.probabilistic:
-        if not comparison.probabilistic:
+        pass
```

Comment:
This currently makes it so that the two if statements below are not run. Is this supposed to happen?

Reply:
forgot to delete that. was a first working demo fix. thats how serious flaws enter the code base.